tests/test_project/settings.py

```diff
@@ -0,0 +1,202 @@
+import os
+import sys
+
+# Import source code dir
+#   This ensures that we can import `resulto_dj.xyz...` from within the tests/
+#   folder containing all the tests from the lib code.
+sys.path.insert(0, os.getcwd())
+sys.path.insert(0, os.path.join(os.getcwd(), os.pardir))
+sys.path.insert(0, os.path.join(os.getcwd(), os.pardir, os.pardir))
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+SECRET_KEY = "abcdef123-"
+
+DEBUG = True
+IN_TEST = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        # XXX This assumes you have the loyalty project setup on your machine
```

Comment:
This was copy-pasted but I don't think you needed postgres here? Should probably remove references to loyalty & resulto_dj